src/routes/GroupExplorer/GroupExplorer.js

```diff
@@ -15,6 +16,10 @@ export default function GroupExplorer ({
 
   return (
     <React.Fragment>
+      <Helmet>
+        <title>Hylo: Explorer</title>
```

Comment:
We should have group name here. Can we build titles piece by piece so we add the group name in the AuthRouter if it exists and then add on Explorer here?

Reply:
It doesn't add titles piece by piece; it takes the description deepest in the hierarchy. So if a group detail page is also rendered, it will grab the meta tags from that, over this one